tests/gdb_lots_of_threads_test.c

```diff
@@ -166,6 +168,13 @@ int main(int argc, char* argv[])
       }
    }
 
+   // Wait for all the threads to be running.
+   // Yes, sometimes the threads have been created but are not yet joinable.
+   while (running_count < i) {
```

Comment:
You want to make this into atomic fetch? Maybe it doesn't matter on x86

Reply:
ok, I change it.